helm/frost-server/README.md

```diff
@@ -154,7 +156,9 @@ The following table lists the configurable parameters of the FROST-Server chart
 |`frost.db.securityContext.fsGroup`          | The fsGroup of the database process.                                                                                                                                                                                                                  | `nil`                                                                                    |
 |`frost.db.requiredHostname`                 | If this is set, an affinity rule will be added, so that the database pod will only be scheduled on the node with this hostname.                                                                                                                       | `nil`
 |`frost.mqtt.enabled`                        | If MQTT support needs to be enabled. See [bellow](#mqtt) for more information                                                                                                                                                                         | `true`                                                                                    |
+|`frost.mqtt.annotations`                        | It is possible to set certain annotation for the service. This can be handy when using `metallb` as a LoadBalancer. With the annotation option it is possible to set a static ip address for the service. | `true`                                                                                    |
 |`frost.mqtt.replicas`                       | The number of FROST-Server MQTT module replicas                                                                                                                                                                                                       | `1`                                                                                       |
+|`frost.mqtt.sericeType`                       | The `serviceType` that Kubernetes will use when deploying the mqtt service. This can either be `NodePort` or `LoadBalancer`. When choosing `LoadBalancer` the service will directly be exposed on a separate ip address by the loadbalancer. This can be useful in cloud environments on if you run `metallb` for example. When using `metallb` you can optionally set a annotation so you can specify a specific ip address to the service.  replicas                                                                                                                                                                                                       | `1`                                                                                       |
```

Comment:
I'd prefer to have `ClusterIp` as default service type. [These](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) should be the allowed values.

Reply:
If using the `ClusterIP` service type, the mqtt is only exposed from within the cluster. Is that the default behavior for the MQTT?